src/service/data_products/common_models.py

```diff
@@ -81,13 +86,13 @@ def _ensure_only_one_view(cls, v):
         for dbc in v:
             if found and v.view_required:
                 raise ValueError("More than one db collection requiring a view found")
-            found = found or dbc.view_required
+            found = found or dbc.view_required or dbc.generic_view_required
         return v
     
     def view_required(self):
         """ Check if a search view is required for this data product. """
         for db in self.db_collections:
-            if db.view_required:
+            if db.view_required or db.generic_view_required:
```

Comment:
This will break updating collections: https://github.com/kbase/collections/blob/24d29cf66451b980472f1e8c6ea8d3dba537d82b/src/service/routes_collections.py#L59

Reply:
:+1: